web/src/drafts_overlay_ui.ts

```diff
@@ -66,8 +115,14 @@ function remove_draft($draft_row: JQuery): void {
     // Deletes the draft and removes it from the list
     const draft_id = $draft_row.attr("data-draft-id")!;
 
+    const draft = drafts.draft_model.getDraft(draft_id);
     drafts.draft_model.deleteDraft(draft_id);
 
+    if (draft) {
+        draft_undo_delete_list.push(draft);
+        show_delete_banner();
```

Comment:
If you delete 20 drafts at once, does `show_delete_banner` get called 20 times, adding/removing the banner repeatedly? 

We can fix this in a follow-up commit or PR, but we should rework that to a bulk operation for the code path that does this in a loop.